frontend/src/components/AppRatingCircle/index.vue

```diff
@@ -0,0 +1,67 @@
+<template>
+    <svg version="1.1" class="svg-loader" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px"
+     y="0px" viewBox="0 0 80 90" xml:space="preserve">
+     <g>
+        <circle stroke="var(--color-light-gray)" fill="none" v-bind:r="r" :cx="cx" :cy="cy"/>
+        <text text-anchor="middle" alignment-baseline="central" x="40" y="40">{{ value }}</text>
+        <circle stroke="var(--theme)" v-bind:r="r" :cx="cx" :cy="cy"
+            :transform="`rotate(-90 ${cx} ${cy})`"
+            :stroke-dasharray="circumfrence"
+            :stroke-dashoffset="rating"/>
+     </g>
+     <text v-if="text" class="description" alignment-baseline="central" x="40" y="80" text-anchor="middle">{{ text }}</text>
+    </svg>
+</template>
+
+<script>
+export default {
+  props: {
+    value: Number,
+    text: String
+  },
+  data() {
+    return {
+      cx: 40,
+      cy: 40,
+      r: 25,
+      rating: 0
+    }
+  },
+  computed: {
+    circumfrence() {
+      return 2 * Math.PI * this.r
+    }
+  },
+  methods: {
+    calcRating() {
+      return ((100 - this.value) / 100) * this.circumfrence
+    }
+  },
+  mounted() {
+    this.rating = this.circumfrence
+    setTimeout(() => {
+      this.rating = this.calcRating()
+    })
+  }
+}
+</script>
+
+<style scoped>
+svg {
+  height: inherit;
+}
+
+circle {
+    fill: none;
+    stroke-width: 5px;
+    transition: stroke-dashoffset 3s ease-in-out;
+}
+
+text {
```

Comment:
needs smol top margin/padding i think; it's too close to the circle rn

Reply:
svg padding isn't really a thing. Might move label out of the component entirely. Thougths?

Reply:
Component would then just be the circle.

Reply:
With the inner text.

Reply:
WIll address this later.

Reply:
Yeah that makes sense. not urgent though